extra.html

```diff
@@ -58,54 +58,45 @@
 
        <kubernetes-topology-icon kind="Pod">
          <svg class="kube-topology">
-           <use xlink:href="#vertex-Pod" x="15" y="15"></use>
+           <g transform="translate(15, 15)" class="Pod">
+             <circle r="15"></circle>
+             <text y="7">P</text>
```

Comment:
> This is the icon in the legend that you're touching here.

I know it is the legend, hence I wrote above that in the legend it seems not to display the png icon too (a white circle). so I'm wondering if that is an issue caused by working with translate. although I used exactly the same x,y and circle radius as before. I'll keep playing with that.